utils/generic-components.stanza

```diff
@@ -266,6 +176,41 @@ public defn look-up-ceramic-caps (attribute: String, filter-properties:Tuple<Key
                                         filter-properties]
   look-up-capacitors(attribute, params)
 
+; ========================================================
+; ==================== tantalum-cap =======================
+; ========================================================
+
+public defn tantalum-cap (user-params:Tuple<KeyValue>) -> Instantiable :
```

Comment:
You can add doc string for public functions to describe the API : 

```
doc: "Create a tantalum cap. Supported key/value args are \"capacitance\", \"tolerance\", etc."
public defn tantalum-cap (user-params:Tuple<KeyValue>) -> Instantiable :

doc: "Create a tantalum cap with given capacitance"
public defn tantalum-cap (cap:Double) -> Instantiable : 

doc: "Create a tantalum cap with a given capacitance and tolerance" 
public defn tantalum-cap (cap:Double, tolerance:Double) -> Instantiable : 
```

The `doc:` syntax is there for stanza's autodocumentation feature. We should start using it more so we can auto-generate OCDB's docs.

Reply:
pass - this is a new doc flow beyond the scope of this change